upgrade/1.2/scripts/upgrade/ncn-upgrade-ceph-nodes.sh

```diff
@@ -158,6 +158,19 @@ fi
 . /usr/share/doc/csm/upgrade/1.2/scripts/ceph/lib/ceph-health.sh
 wait_for_health_ok
 
+# Wait for rgw to start before executing goss tests
+target_ncn=ncn-s001
+rgw_counter=0
+until [[ $(ceph orch ps --daemon_type rgw ${target_ncn} --format json-pretty|jq -r '.[].status_desc') == "running" ]]
+do
+  sleep 30
+  let rgw_counter+=1
+  if rgw_counter -gt 10
```

Comment:
Does this need to be between [[ ]] ?
```suggestion
  if [[ rgw_counter -gt 10 ]]
```